woocommerce/woocommerce
# Update ImageGallery toolbar: changing icon to text and adding ellipsis menu

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Update ImageGallery toolbar: changing icon to text and adding ellipsis menu

Closes #38954 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Load this branch and go to **Settings > Advanced > Features** and enable the **New Product Editor**
2. Click **Products > Add New** and scroll down to the **Images** field
3. Click **Choose images** and select several images ( you can hold shift to select several )
4. Once added, click on a none-cover image. Notice how the toolbar has a `Set as cover` button
5. Click the `Set as cover` button, and it should replace the first (cover ) image with this one.
6. Now click on another none-cover image and click the 3-dot menu
7. You should see the Replace and Remove options
8. Click **Replace** and select an image to see if it still correctly replaces the image
9. Click **Remove** to make sure it still correctly removes the image.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
